lib/api/2.0/nodes.js

```diff
@@ -61,26 +60,7 @@ var nodesGetPollersById = controller(function(req) {
 });
 
 var nodesGetWorkflowById = controller(function(req) {
-    var newQuery;
```

Comment:
Removing this code breaks the `?active=true` and `?active=false` query parameters.  These were explicitly changed from the /active endpoint in the 1.1 API because, at the time, there was some confusion about the semantics.  Adding the query makes things more explicit.